lightspark-crypto-go/signing.go

```diff
@@ -4,8 +4,8 @@ package lightspark_crypto
 
 // #cgo darwin,amd64 LDFLAGS: -L${SRCDIR}/libs/darwin/amd64 -llightspark_crypto
 // #cgo darwin,arm64 LDFLAGS: -L${SRCDIR}/libs/darwin/arm64 -llightspark_crypto
-// #cgo linux,amd64 LDFLAGS: -L${SRCDIR}/libs/linux/amd64 -llightspark_crypto
-// #cgo linux,arm64 LDFLAGS: -L${SRCDIR}/libs/linux/arm64 -llightspark_crypto
+// #cgo linux,amd64 LDFLAGS: -L${SRCDIR}/libs/linux/amd64 -Wl,-Bstatic -llightspark_crypto -Wl,-Bdynamic
```

Comment:
This basically tells the linker to link `lightspark_crypto` statically but continue to dynamically link everything else.